src/components/Header/index.jsx

```diff
@@ -8,42 +8,53 @@ import LinkWrapper from '../LinkWrapper';
 import EdxLogo from '../../images/edx-logo.png';
 import './Header.scss';
 
-const Header = (props) => {
-  const {
-    enterpriseLogo,
-    email,
-  } = props;
+class Header extends React.Component {
+  renderLogo() {
+    const { enterpriseLogo, enterpriseName } = this.props;
+    return (
+      <Img
+        src={enterpriseLogo || EdxLogo}
+        alt={`${enterpriseName || 'edX'} logo`}
+        onError={(e) => { e.target.src = EdxLogo; }}
```

Comment:
Similar to the logic in the `Footer` above, but in the callback function for `onerror` we _do_ want to show a logo (otherwise, the `Header` would be logoless!). In this case, if the enterprise logo can't be loaded, we swap out the `img` src with the edX logo.